code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -0,0 +1,518 @@
+		// Greys get to keep their implant
+		if(isgrey(H) && istype(I, /obj/item/organ/internal/cyberimp/brain/speech_translator))
+			continue
+		// Try removing it
+		I = I.remove(H)
+		if(I)
+			stuff_to_transfer += I
+
+	// Transfer it all (or drop it if not possible)
+	for(var/i in stuff_to_transfer)
+		var/obj/item/I = i
+		if(GLOB.prisoner_belongings.give_item(I))
+			victim_belongings += I
+		else if(!((ABSTRACT|NODROP) in I.flags)) // Anything that can't be put on hold, just drop it on the ground
+			I.forceMove(T)
+
+	// Give some species the necessary to survive. Courtesy of the Syndicate.
+	if(istype(H))
+		var/obj/item/tank/emergency_oxygen/tank
+		var/obj/item/clothing/mask/breath/mask
+		if(isvox(H))
+			tank = new /obj/item/tank/emergency_oxygen/nitrogen(H)
+			mask = new /obj/item/clothing/mask/breath/vox(H)
+		else if(isplasmaman(H))
+			tank = new /obj/item/tank/emergency_oxygen/plasma(H)
+			mask = new /obj/item/clothing/mask/breath(H)
+
+		if(tank)
+			H.equip_to_appropriate_slot(tank)
+			H.equip_to_appropriate_slot(mask)
+			tank.toggle_internals(H, TRUE)
+
+	M.update_icons()
+
+	// Narrate their kidnapping and torturing experience.
+	if (M.stat != DEAD)
+		// Heal them up - gets them out of crit/soft crit.
+		M.reagents.add_reagent("omnizine", 20)
+
+		to_chat(M, "<span class='warning'>You feel strange...</span>")
+		M.Paralyse(30 SECONDS_TO_LIFE_CYCLES)
+		M.EyeBlind(35 SECONDS_TO_LIFE_CYCLES)
+		M.EyeBlurry(35 SECONDS_TO_LIFE_CYCLES)
+		M.AdjustConfused(35 SECONDS_TO_LIFE_CYCLES)
+
+		sleep(6 SECONDS)
+		to_chat(M, "<span class='warning'>That portal did something to you...</span>")
+
+		sleep(6.5 SECONDS)
+		to_chat(M, "<span class='warning'>Your head pounds... It feels like it's going to burst out your skull!</span>")
+
+		sleep(3 SECONDS)
+		to_chat(M, "<span class='warning'>Your head pounds...</span>")
+
+		sleep(10 SECONDS)
+		to_chat(M, "<span class='specialnotice'>A million voices echo in your head... <i>\"Your mind held many valuable secrets - \
+					we thank you for providing them. Your value is expended, and you will be ransomed back to your station. We always get paid, \
+					so it's only a matter of time before we send you back...\"</i></span>")
+
+/**
+  * Handles the target's return to station.
+  *
+  * Arguments:
+  * * M - The target mob.
+  * * T - The turf the target was extracted from.
+  */
+/datum/syndicate_contract/proc/handle_target_return(mob/living/M, turf/extraction_turf)
+	var/list/turf/possible_turfs = list()
+	for(var/turf/T in (list(extraction_turf) + contract.extraction_zone.contents))
+		if(!isspaceturf(T) && !isunsimulatedturf(T) && !is_blocked_turf(T))
+			possible_turfs += T
+
+	var/turf/destination = length(possible_turfs) ? pick(possible_turfs) : pick(GLOB.latejoin)
+
+	// Return their items
+	for(var/i in victim_belongings)
+		var/obj/item/I = GLOB.prisoner_belongings.remove_item(i)
+		if(!I)
+			continue
+		I.forceMove(destination)
```

Comment:
I wonder if we should put the items (and the victim) into a fancy syndicate locker, instead of just dropping them on the ground...

Reply:
I'm not sure.. having a syndicate locker sent back would be a telltale sign that there is a contractor about, which is bad if they're been so far undetected. Perhaps a regular locker?

Reply:
I don't have a strong opinion. Regular locker (or a crate even) should work.